Autopilot/Inc/IMU.hpp

```diff
@@ -1,25 +1,31 @@
 /**
  * IMU Sensor Functions and Part Number Selection
- * Author: Lucy Gong
+ * Author(s): Lucy Gong, Dhruv Rawat
  */
 
 #ifndef IMU_HPP
 #define IMU_HPP
 
+#include <stdlib.h>
+#include <ctime>
+#include <cstdint>
+#include <math.h>
+
 #define ICM_20602 0
 #define MPU9255 1
 
 #define USE_IMU ICM_20602
 
 struct IMUData_t {
 
-    float magx, magy, magz;
+    float magx, magy, magz; //ICM20602 does not have a magnetometer. Thus, these readings will be left as NAN
     float accx, accy, accz;
     float gyrx, gyry, gyrz; 
+    float temp; 
 
     bool isDataNew; 
     int sensorStatus; //TBD but probably 0 = SUCCESS, -1 = FAIL, 1 = BUSY 
```

Comment:
More of a quirk, but I think an enum would be better suited for states than integers defined in comments imo

Reply:
We'll have to decide on a consistent style for this before we change this so all drivers return similarly structured data.